client/jetpack-cloud/components/sidebar/style.scss

```diff
@@ -10,6 +10,20 @@
 		background-color: var(--color-sidebar-background);
 	}
 
+
+	.split-button {
```

Comment:
Non-blocker, Maybe we can set the `a` element width to `calc(70% - 50px)` or display the split button as a flexbox and have `a` element grow (`flex-grow: 1`). This way, the toggle button will have a fixed width and won't look too wide, in my opinion.

Reply:
Agree. Done: https://github.com/Automattic/wp-calypso/pull/83455/commits/60c20305aadf3b53856bafb818c005c767fcccb2